simple_trajectories/src/main.cpp

```diff
@@ -76,16 +76,11 @@ void moveArmTo(
   armSpace->convertStateToPositions(state, positions);
   ROS_INFO_STREAM(positions.transpose());
 
-  auto smoothTrajectory = robot.postProcessPath<ParabolicSmoother>(
-      trajectory.get(), satisfied, ParabolicSmoother::Params());
-  aikido::trajectory::TrajectoryPtr timedTrajectory
-      = std::move(robot.postProcessPath<KunzRetimer>(
-          smoothTrajectory.get(), satisfied, ada::KunzParams()));
-
   waitForUser("Press key to move arm to goal");
-  auto future = robot.executeTrajectory(timedTrajectory);
+  CollisionFreePtr collisionFree = robot.getSelfCollisionConstraint(
+      robot.getArm()->getStateSpace(), armSkeleton);
```

Comment:
Using `armSpace` doesn't work here as it returns a failure about the collision being outside of the stateSpace. I don't quite understand what the differences between the `robot` stateSpace, the arm stateSpace, and the metaSkeleton stateSpace, but only the arm stateSpace seems to work for the collision constraint construction.

I noticed a lot of call sites passing a `nullptr` for `CollisionFreePtr`, that also works here.